easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,148 @@
+name = 'Ruby'
+version = '2.7.1'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with
+ a focus on simplicity and productivity. It has an elegant syntax that is
+ natural to read and easy to write."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://cache.ruby-lang.org/pub/ruby/%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['d418483bdd0000576c1370571121a6eb24582116db0b7bb2005e90e250eae418']
+
+builddependencies = [('binutils', '2.32')]
+
+exts_default_options = {
+    'source_urls': ['https://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
```

Comment:
I would a comment explaining some guidelines for this extension list to be known in the future
```suggestion

# !! order of packages is important !!
# some packages have dependencies with minimum and maximum version requirements
# each version is picked as high as possible to fullfill all requirements
# packages updated on 8 April 2020
```